pageserver/src/tenant_mgr.rs

```diff
@@ -137,13 +135,14 @@ pub fn register_relish_download(
 
     {
         let mut m = access_tenants();
-        let mut tenant = m.get_mut(&tenant_id).unwrap();
+        let tenant = m.entry(tenant_id).or_insert_with(|| Tenant {
+            state: TenantState::Downloading,
+            repo: None,
+        });
         tenant.state = TenantState::Downloading;
```

Comment:
Does a downloaded tenant get marked as active later?

Reply:
It does not matter at this stage much, the whole `tenant.state = TenantState::Downloading;` should go elsewhere, since we don't really download anything at the moment we register the freshly downloaded image.

But sure, I've added a status change.
Proper fixes to this should come in https://github.com/zenithdb/zenith/issues/756